lib/SampleService/core/samples.py

```diff
@@ -186,6 +186,17 @@ def get_sample(
         self._check_perms(_not_falsy(id_, 'id_'), user, _SampleAccessType.READ, as_admin=as_admin)
         return self._storage.get_sample(id_, version)
 
+    def get_samples(
```

Comment:
same question here. I think by reusing `get_sample` here, we can save all updates in `arango_sample_storage.py`.